CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Changelog
 
+### v1.0.1 Oct 4, 2018
+
+* fixed iOS Location->currencyCode conversion
```

Comment:
We should wrap code objects in code ticks to clarify when "->" is being used to denote a transition and note part of a code block.